src/PIL/Image.py

```diff
@@ -1929,11 +1923,23 @@ def save(self, fp, format=None, **params):
 
         if format.upper() not in SAVE:
             init()
+        save_all = False
+        if 'save_all' in params:
+            save_all = params.pop('save_all')
```

Comment:
This could be `save_all = params.pop('save_all', False)`

Reply:
Good call, thanks. I've made that change.